py/desisim/lya_spectra.py

```diff
@@ -128,14 +128,17 @@ def read_lya_skewers(lyafile,indices=None,read_dlas=False,add_metals=False,add_l
           if "F_METALS" in h:
               metals = h["F_METALS"].read()
               trans *= metals
+              log.info('Added F_Metals from file')
           #For format london v<7.3
-          if "METALS" in h :
+          elif "METALS" in h :
               metals = h["METALS"].read()
               trans *= metals
-          else :
+              log.info('Added Metals from file')
+          else:
               nom="No HDU with EXTNAME='METALS' or EXTNAME='F_METALS' in transmission file {}".format(lyafile)
```

Comment:
THis this an error, it shoud start with `ERROR: `, so can be cathed with a `grep`

Reply:
This is added by the line: raise KeyError(nom). For instance this is the message when it runs in a previous version. Let me know if this is sufficient, all errors/warning/info is added this way...
```

WARNING:quickquasars.py:783:main: --mags is deprecated; please use --bbflux instead
INFO:quickquasars.py:811:main: Load SIMQSO model
WARNING:templates.py:2361:__init__: Using modified simqso.sqmodels defined in desisim.scripts.lya_simqso_model
INFO:quickquasars.py:819:main: Load DeCAM and WISE filters for target selection sim.
ifilename /project/projectdirs/desi/mocks/lya_forest/develop/london/v7.3/v7.3.0/0/0/transmission-16-0.fits
INFO:quickquasars.py:205:get_healpix_info: healpix=HPXPIXEL=0
INFO:quickquasars.py:210:get_healpix_info: nside=HPXNSIDE=16
INFO:quickquasars.py:218:get_healpix_info: hpxnest from HPXNEST = True
INFO:quickquasars.py:297:simulate_one_healpix: Read skewers in /project/projectdirs/desi/mocks/lya_forest/develop/london/v7.3/v7.3.0/0/0/transmission-16-0.fits, random seed = 2866
INFO:quickquasars.py:301:simulate_one_healpix: Read transmission file /project/projectdirs/desi/mocks/lya_forest/develop/london/v7.3/v7.3.0/0/0/transmission-16-0.fits
ERROR:lya_spectra.py:139:read_lya_skewers: No HDU with EXTNAME='METALS' or EXTNAME='F_METALS' in transmission file /project/projectdirs/desi/mocks/lya_forest/develop/london/v7.3/v7.3.0/0/0/transmission-16-0.fits
Traceback (most recent call last):
  File "/global/homes/a/alxogm/desi/code/desisim/bin/quickquasars", line 9, in <module>
    sys.exit(main())
  File "/global/homes/a/alxogm/desi/code/desisim/py/desisim/scripts/quickquasars.py", line 876, in main
    bal=bal,sfdmap=sfdmap,eboss=eboss)
  File "/global/homes/a/alxogm/desi/code/desisim/py/desisim/scripts/quickquasars.py", line 303, in simulate_one_healpix
    trans_wave, transmission, metadata, dla_info = read_lya_skewers(ifilename,read_dlas=(args.dla=='file'),add_metals=args.metals_from_file,add_lyb=args.add_LYB)
  File "/global/homes/a/alxogm/desi/code/desisim/py/desisim/lya_spectra.py", line 140, in read_lya_skewers
    raise KeyError(nom)
KeyError: "No HDU with EXTNAME='METALS' or EXTNAME='F_METALS' in transmission file /project/projectdirs/desi/mocks/lya_forest/develop/london/v7.3/v7.3.0/0/0/transmission-16-0.fits"
```

Reply:
perfect.